cocos/renderer/CCRenderer.cpp

```diff
@@ -809,6 +829,11 @@ void Renderer::drawBatchedTriangles()
         // so most probably we won't have any benefit of using it
         glBufferData(GL_ARRAY_BUFFER, sizeof(_verts[0]) * _filledVertex, nullptr, GL_STATIC_DRAW);
         void *buf = glMapBuffer(GL_ARRAY_BUFFER, GL_WRITE_ONLY);
+        GLenum err = glGetError();
+        if (err != GL_NO_ERROR)
+        {
+            UserDefault::getInstance()->setBoolForKey("IsExitInDrawBatchedTriangle", true);
```

Comment:
Use expensive IO operations (even without a "modern" version with expensive mutexes) during GPU rendering loop is unexceptable.

Reply:
@gestern I know, this will be removed. It was just added for some tests I did, it obviously should not be merged.